### PR TITLE
[NSDK-196] Update README for Jetpack Compose UI Components

### PR DESCRIPTION
**Pull request by akueisara**
## 🔗 Related Links

- [x] [ClickUp](https://app.clickup.com/t/3702259/NSDK-196)

## ➡️ To Be

- [x] Created README-COMPOSE.md
- [x] Modified README.md
- [x] Updated the sample code 
- [x] Change the default colors of VirtusizeButton 

## ☑️ Checklist

- [ ] Useless comments and/or `Log.d` etc are **removed**
- [ ] Unit test are **covered**
- [ ] Code has been **deployed and tested** on STG
- [x] ClickUp ticket status has been **updated** to `DEV REVIEW`
- [x] Update CHANGELOG.md with the new changes

### CODE REVIEW

**Review by ourognao — COMMENTED** [review]
@akueisara 2 comments

File: sampleAppKotlin/src/main/java/com/virtusize/sampleappkotlin/App.kt [hunk 1]
```diff
@@ -1,35 +1,31 @@
...
+            // Only the API key is required
+            .setApiKey("15cc36e1d7dad62b8e11722ce1a245cb6c5e6692")
+            // For using the Order API, a user ID is required
+            .setUserId("123")
+            // By default, the Virtusize environment will be set to GLOBAL
+            .setEnv(VirtusizeEnvironment.JAPAN)
+            // By default, the initial language will be set based on the Virtusize environment
+            .setLanguage(VirtusizeLanguage.EN)
+            // By default, ShowSGI is false
+            .setShowSGI(true)
```
[the remark this answers is not in the document]
Comment (akueisara) [reply sampleAppKotlin/src/main/java/com/virtusize/sampleappkotlin/App.kt:24]: Without setting ShowSGI, it's set to false.

[the remark this answers is not in the document]
Comment (akueisara) [reply sampleAppKotlin/src/main/java/com/virtusize/sampleappkotlin/App.kt:24]: Without setting showSGI, the default value is false. The example here is to set it as true.

File: README.md [hunk 2]
```diff
@@ -310,9 +309,9 @@ same guidelines in the case of fragment as well.
   ```kotlin
   private val activityMessageHandler: VirtusizeMessageHandler
   override fun onPause() {
-          // Always un register message handler in onPause() or depending on implementation onStop().
-          (application as App).virtusize.unregisterMessageHandler(activityMessageHandler)
-          super.onPause()
+      // Always un register message handler in onPause() or depending on implementation onStop().
```
[the remark this answers is not in the document]
Comment (ourognao) [reply README.md]: Verified thanks

File: sampleAppKotlin/src/main/java/com/virtusize/sampleappkotlin/App.kt (hunk 1 above)
[the remark this answers is not in the document]
Comment (ourognao) [reply sampleAppKotlin/src/main/java/com/virtusize/sampleappkotlin/App.kt:24]: I see. It was confusing.

**Review by ourognao — APPROVED** [review]
cat-shadow

**Pull request merged**